airflow/contrib/kubernetes/worker_configuration.py

```diff
@@ -183,6 +229,28 @@ def _construct_volume(name, claim, host):
             del volumes[self.dags_volume_name]
             del volume_mounts[self.dags_volume_name]
 
+        # Get the SSH key from secrets as a volume
+        if self.kube_config.git_ssh_key_secret_name:
+            volumes[self.git_sync_ssh_secret_volume_name] = {
+                'name': self.git_sync_ssh_secret_volume_name,
+                'secret': {
+                    'secretName': self.kube_config.git_ssh_key_secret_name,
+                    'items': [{
+                        'key': self.git_ssh_key_secret_key,
+                        'path': 'ssh',
+                        'mode': 288
```

Comment:
Little bit easier to understand the mode if we use octal:

```suggestion
                        'mode': 0o440
```